tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -634,7 +635,10 @@ def fit(self,
                 to any amount of labeled data.
             verbosity : int
                 If specified, overrides the existing `predictor.verbosity` value.
-
+            calibrate: bool, default = False
+                If True then will use temperature scaling to calibrate the model for better negative log loss. Temperature scaling
```

Comment:
"the model for better negative log loss" -> 
"the Predictor's estimated class probabilities in classification tasks (which may improve metrics like `log_loss`)"